buildpacks/java/tests/build_test.go

```diff
@@ -0,0 +1,162 @@
+// Copyright 2021-2022 VMware, Inc.
+// SPDX-License-Identifier: BSD-2-Clause
+
+package tests
+
+import (
+	"os"
+	"testing"
+
+	"github.com/buildpacks/libcnb"
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+
+	. "github.com/onsi/gomega"
+
+	"kn-fn/java-function-buildpack/java"
+)
+
+const (
+	HTTPFuncTemplate = `from typing import Any
```

Comment:
This also looks like it's for python, not java. I would expect the tests to fail because of this, so it might be something to look into.

Reply:
+1 why is there Python?